app/helpers/sessions_helper.rb

```diff
@@ -42,4 +42,25 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  # Returns true if the given user is the current user.
+  def current_user? user
+    user && user == current_user
```

Comment:
```suggestion
    user == current_user
```

Chỗ này ko cần phải kiếm tra user đâu ông. nil thì nó cx false thôi